app/services/place_hold_results/builder.rb

```diff
@@ -21,18 +21,10 @@ def generate
   private
 
     def hold_lookup(hold_key)
-      parsed_hold = SymphonyClientParser::parsed_response(@client, :get_hold_info, hold_key, @user_token)
-
-      start = DateTime.now
-      while parsed_hold.dig('fields', 'item').nil? && start + 5.seconds > DateTime.now
```

Comment:
per our conversation, we need this item level info check for the response, guess we might add to the new SirsiResponse as a method like similar to `busy?`